File: src/main/java/de/maggiwuerze/xdccloader/events/server/ServerDoneEvent.java
package de.maggiwuerze.xdccloader.events.server;

import de.maggiwuerze.xdccloader.events.EntityDoneEvent;
import de.maggiwuerze.xdccloader.model.entity.Server;

public class ServerDoneEvent extends EntityDoneEvent<Server> {

	public ServerDoneEvent(Object source, Server server) {
		super(source, server);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/server/ServerDoneEvent.kt
package de.maggiwuerze.xdccwebloader.events.server

import de.maggiwuerze.xdccwebloader.events.EntityDoneEvent
import de.maggiwuerze.xdccwebloader.model.entity.Server

class ServerDoneEvent(source: Any, server: Server) : EntityDoneEvent<Server?>(source, server)